tests/case/case.php

```diff
@@ -231,6 +231,21 @@
     {
         // Nothing
     }
+
+    public function parentTest()
+    {
+        $array = [];
+        echo $array[iNt];
+        echo $array[sElF];
+        echo $array[pArEnT];
+        echo $array[aRaRay];
```

Comment:
is `aRaRay` meant to be `array` with a different case?

Reply:
sure, but since the other constants (`iNt`, `sElF`, `pArEnT`) looked like keywords (`int`, `self`, `parent`), I wondered if `aRaRay` was also meant to

Reply:
@nicoder it is also constant 
http://php.net/manual/en/reserved.keywords.php vs http://php.net/manual/en/reserved.other-reserved-words.php :trollface: php very interesting language

Reply:
I think the point was that `array` is spelt wrong.

Reply:
@aboyton just run:
```
define('aRaRay', 'foo');

$array = [aRaRay => 1];
echo $array[aRaRay];
```